.github/workflows/main.yml

```
@@ -154,7 +156,7 @@ jobs:
   tests:
     name: tests-python${{ matrix.python-version }}-${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    needs: [standalone, wheel, sdist, lint, docs]
+    needs: [standalone, sdist, lint, docs]
```

We're still having job timeout issues despite the timeout fixtures and test performance improvements. Something else must be going on. But until we get to the bottom of it we should timeout sooner than 360mins after a job starts. Lets add:

```suggestion
    needs: [standalone, sdist, lint, docs]
    timeout-minutes: 25
```